bibliopixel/animation/animation.py

```diff
@@ -119,7 +119,8 @@ def run_context(self):
 
         self.check_delay()
 
-        self.preRun(self.runner.amt)
+        self.layout.all_off()
```

Comment:
Pulling up the `all_off()` to here makes me happy - but I'm not entirely sure it's correct.

What if my animation perturbs the previous state of the layout?  Typical example - I want an animation that fades in.